storage/src/main/java/tech/pegasys/teku/storage/server/rocksdb/RocksDbDatabase.java

```diff
@@ -207,7 +210,10 @@ public void storeInitialAnchor(final AnchorPoint anchor) {
       // We need to store the anchor block in both hot and cold storage so that on restart
       // we're guaranteed to have at least one block / state to load into RecentChainData.
       // Save to hot storage
-      hotUpdater.addHotBlock(anchorBlock);
+      hotUpdater.addHotBlock(
+          new BlockAndCheckpointEpochs(
+              anchorBlock,
+              new CheckpointEpochs(anchorCheckpoint.getEpoch(), anchorCheckpoint.getEpoch())));
```

Comment:
I think this should be:
```suggestion
              new CheckpointEpochs(anchorState.getCurrent_justified_checkpoint().getEpoch(), anchorState.getFinalized_checkpoint().getEpoch())));
```